*add advanced store *reorganize sections *add icon,favicon (#53)
merantix-momentum/squirrel-core: docs/usage/store.rst

    driver = MessagepackDriver(tmpdir.name)
    store = driver.store

> Sharding

Reviewer: ```suggestion
Sample and Shard
```

--------

> :py:class:`squirrel.store.SquirrelStore` uses a concept called sharding to efficiently store and load data.

Reviewer: here please say a sentence about what a sample is. something like this: Sample is any python object, usually a dictionary, representing a single training sample for model training. It may include a numpy array and basic data types.

> Shards stores a predetermined amount of samples given through the shard size in a fixed order.

Reviewer: ```suggestion
Shard is a collection of samples, it stores a predetermined number of samples in a fixed order.
```

Each shard is then identified through a unique key. A sample is of type :py:class:`Dict[str, Any]` and a shard is a
list thereof i.e. :py:class:`List[Dict[str, Any]]`.

> 

Reviewer: I suggest to move this up to line 39, sample and shard is mentioned there.

Author: I don't see any mentions of samples and shards at line.39? Is is where SquirrelStore is introduced?
